velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -434,6 +434,22 @@ TEST_P(PrestoSerializerTest, ioBufRoundTrip) {
   }
 }
 
+TEST_P(PrestoSerializerTest, roundTrip) {
```

Comment:
A fuzzer-based test is nice, but it would be helpful to also add a test case that specifically reproduces the bug being fixed.